src/darwin/Framework/CHIP/CHIPOperationalCredentialsDelegate.mm

```diff
@@ -100,6 +101,12 @@ static BOOL isRunningTests(void)
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CHIPOperationalCredentialsDelegate::GetDeviceCATs(VendorId vid, uint16_t pid, CATValues & cats)
```

Comment:
How would a framework consumer actually use this?  The answer is ... they can't.  We should, imo, be honest and just inline this into the callsite, until we figure out what API will actually be useful here for the framework.

Reply:
I removed this logic from the framework and will create a separate ticket to define and implement new APIs to set CAT values.